### ISSUE TITLE
Not inviting, script running.

### ISSUE THREAD

**Issue by MAPReiff** [mandreasen/SteamAutomaticGroupInvite#1]
Hello!
So I recently installed your script only to find out that it does not function. When loading a steam profile, I see the little 1 next to the tamper monkey symbol symbolizing that it is running, but the user does not receive an invite. It has been tested running on the group owner, moderators and normal members with no luck. I have tried the following in the config, none work:

var steam_group_custom_url = "steamcommunity.com/groups/tradingllc";
var steam_group_custom_url = "tradingllc";
var steam_group_custom_url = steamcommunity.com/groups/tradingllc;
var steam_group_custom_url = tradingllc;

Am I doing it wrong?
Tamper money version - 4.0.69
Chrome Version - 50.0.2661.102
OS - Windows 10 Home 64bit (Also tried Mac OSX 10.11.4 with no luck).

**Comment by mandreasen** [comment]
Hi, There.

Can you copy your console log from your browser. Press F12 and click on Console in the little window.

Because i have just tested it and it still work fine for me. Your need to have invite access in the group else Steam will send you a 403 error.

If the text is your console says "[STEAMID64] Invite to Join Your Group." or "[STEAMID64] The user are already in the group or have already received invites." the script is running.

So it should work if you have invite access and set line 16 to: var steam_group_custom_url = "tradingllc";

**Comment by MAPReiff** [comment]
Ok, so trying to hit F12 did not pull up the console (neither did FN + F12), however, I assume the console in the in the inspect element must be similar or the same. I get the following

```
VM888:9 Mixed Content: The page at 'https://steamcommunity.com/id/comixsoffshore' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://steamcommunity.com/groups/tradingllc/memberslistxml?xml=1'. This request has been blocked; the content must be served over HTTPS.k.cors.a.crossDomain.send @ VM888:9n.extend.ajax @ VM888:9GetGroupData @ VM888:56(anonymous function) @ VM888:69(anonymous function) @ VM827:2(anonymous function) @ VM888:4(anonymous function) @ VM888:5(anonymous function) @ VM827:2v @ VM852:52(anonymous function) @ VM887:1(anonymous function) @ VM827:2b @ VM827:1ha @ VM852:54C @ VM852:52Pa.create @ VM852:61x @ VM852:13
VM888:9 XMLHttpRequest cannot load http://steamcommunity.com/groups/tradingllc/memberslistxml?xml=1. Failed to start loading.k.cors.a.crossDomain.send @ VM888:9n.extend.ajax @ VM888:9GetGroupData @ VM888:56(anonymous function) @ VM888:69(anonymous function) @ VM827:2(anonymous function) @ VM888:4(anonymous function) @ VM888:5(anonymous function) @ VM827:2v @ VM852:52(anonymous function) @ VM887:1(anonymous function) @ VM827:2b @ VM827:1ha @ VM852:54C @ VM852:52Pa.create @ VM852:61x @ VM852:13
VM888:64 The request failed or the group custom URL is wrong.
```

Is it because I am connecting over HTTPS like it metions?

**Comment by MAPReiff** [comment]
Ok, just tested it without https, and it works. Is there a way to allow it to function with https?

**Comment by mandreasen** [comment]
Hi.

I will try to fix the https problem thanks for reporting it.

**Comment by MAPReiff** [comment]
I actually fixed it myself, I can try and contribute it right now.
EDIT - won't let me contribute, lines 29 & 47 need to be changed from HTTP to HTTPS.

**Comment by mandreasen** [comment]
Hi CoolComics.

The script is working with HTTP and HTTPS now and I have uploaded the fix (Download version 1.0.1)
